src/DssSpell.sol

```diff
@@ -49,59 +50,35 @@ contract DssSpellAction is DssAction {
-    address constant UNIV2DAIUSDT_JOIN  = 0xAf034D882169328CAf43b823a4083dABC7EEE0F4;
-    address constant UNIV2DAIUSDT_FLIP  = 0xD32f8B8aDbE331eC0CfADa9cfDbc537619622cFe;
-    address constant UNIV2DAIUSDT_PIP   = 0x69562A7812830E6854Ffc50b992c2AA861D5C2d3;
+    address constant LERP = 0x7b3799b30f268BA55f926d7F714a3001aF89d359;
+
+    // Turn off office hours
+    function officeHours() public override returns (bool) {
+        return false;
+    }
 
     function actions() public override {
-        // Rates Proposal - February 22, 2021
-        DssExecLib.setIlkStabilityFee("ETH-A", FIVE_PT_FIVE_PCT, true);
-        DssExecLib.setIlkStabilityFee("ETH-B", NINE_PCT, true);
-
-        // Onboard UNIV2DAIUSDT-A
-        DssExecLib.addReaderToMedianWhitelist(
-            LPOracle(UNIV2DAIUSDT_PIP).orb1(),
-            UNIV2DAIUSDT_PIP
-        );
-        CollateralOpts memory UNIV2DAIUSDT_A = CollateralOpts({
-            ilk: "UNIV2DAIUSDT-A",
-            gem: UNIV2DAIUSDT_GEM,
-            join: UNIV2DAIUSDT_JOIN,
-            flip: UNIV2DAIUSDT_FLIP,
-            pip: UNIV2DAIUSDT_PIP,
-            isLiquidatable: true,
-            isOSM: true,
-            whitelistOSM: false,
-            ilkDebtCeiling: 3 * MILLION,
-            minVaultAmount: 2000,
-            maxLiquidationAmount: 50000,
-            liquidationPenalty: 1300,
-            ilkStabilityFee: FOUR_PCT,
-            bidIncrease: 300,
-            bidDuration: 6 hours,
-            auctionDuration: 6 hours,
-            liquidationRatio: 12500
-        });
-        addNewCollateral(UNIV2DAIUSDT_A);
-        DssExecLib.setChangelogAddress("UNIV2DAIUSDT",             UNIV2DAIUSDT_GEM);
-        DssExecLib.setChangelogAddress("MCD_JOIN_UNIV2DAIUSDT_A",  UNIV2DAIUSDT_JOIN);
-        DssExecLib.setChangelogAddress("MCD_FLIP_UNIV2DAIUSDT_A",  UNIV2DAIUSDT_FLIP);
-        DssExecLib.setChangelogAddress("PIP_UNIV2DAIUSDT",         UNIV2DAIUSDT_PIP);
-
-        // Lower PSM-USDC-A Toll Out
-        Fileable(DssExecLib.getChangelogAddress("MCD_PSM_USDC_A")).file("tout", 4 * WAD / 10000);
-
-        // bump Changelog version
-        DssExecLib.setChangelogVersion("1.2.8");
+        VatAbstract vat = VatAbstract(DssExecLib.vat());
+
+        // De-authorize the lerp contract from adjusting the PSM-USDC-A DC
+        DssExecLib.deauthorize(address(vat), LERP);
+
+        // Increase PSM-USDC-A to 1 Billion from its current value.
+        DssExecLib.setIlkDebtCeiling("PSM-USDC-A", 1000 * MILLION);
+
+        // Decrease the USDC-A Debt Ceiling to zero from its current value.
+        (,,,uint256 line,) = vat.ilks("USDC-A");
+        DssExecLib.setIlkDebtCeiling("USDC-A", 0);
+
+        // Global debt ceiling for PSM was previously set to the end lerp value of 500M
+        // Increase it by another 500M to match the 1B target debt ceiling
+        // Also subtract out the USDC-A line
+        vat.file("Line", vat.Line() + 500 * MILLION * RAD - line);
```

Comment:
I guess it's not really necessary, but maybe add the parens around `(500 * MILLION * RAD)` so that the intent and order of operations is clearer.